kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2718,9 +2728,7 @@ object KyuubiConf {
   val SERVER_ADMINISTRATORS: ConfigEntry[Set[String]] =
     buildConf("kyuubi.server.administrators")
       .doc("Comma-separated list of Kyuubi service administrators. " +
-        "We use this config to grant admin permission to any service accounts when " +
-        s"security mechanism is enabled. Note, when ${AUTHENTICATION_METHOD.key} is " +
-        "configured to NOSASL or NONE, everyone is treated as administrator.")
+        "We use this config to grant admin permission to any service accounts.")
```

Comment:
why change this?

Reply:
It should be a mistake. Thank you for reminding me. I will fix it immediately.